### PR TITLE
Felipe add patient overview

### PR DESCRIPTION
**Pull request by fcoralsasso**

### CODE REVIEW

File: src/components/Graphs/CustomOfflineChart.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      },
+      chart: {
+        plotBackgroundColor: null,
+        plotBorderWidth: null,
+        plotShadow: false,
+        type: this.props.chartType,
+        height: this.props.height,
+      },
+      title: {
+        text: "Distribuition of " + this.splitString(this.props.barTitle),
```
Comment (haoyuanli) [inline src/components/Graphs/CustomOfflineChart.js:60]: typo

Comment (fcoralsasso) [reply src/components/Graphs/CustomOfflineChart.js]: Done.

File: src/layouts/Admin.js [hunk 2]
```diff
@@ -45,10 +48,29 @@ class Dashboard extends React.Component {
...
 
   updateState = (values) => {
-    this.setState(values)
+    this.setState(values);
+  };
+
+  fetchData(datasetId) {
+  
+    if (datasetId) {
+      console.log("request made")
```
Comment (haoyuanli) [inline src/layouts/Admin.js:61]: unneeded console statement

Comment (fcoralsasso) [reply src/layouts/Admin.js]: Done.

**Pull request merged**